### ISSUE TITLE
Uncaught Error: Texture Error: frame does not fit inside the base Texture dimensions [object Object]

### ISSUE THREAD

**Issue by jorgen-17** [beeglebug/pixi-tiled#9]
Hey I ran into this error when using this project. Really cool idea btw. But yeah I'm new at tilemaps and game dev in general. Not sure if Im not setting up my tilemaps correctly. 

its not a 404 issue, it finding the json file and its finding the textures np. This is my tilemap: 

`````` javascript
{ "height":10,
 "layers":[
        {
         "compression":"zlib",
         "data":"eJxjZGBgYBzFgwYDAFB4AGU=",
         "encoding":"base64",
         "height":10,
         "name":"Tile Layer 1",
         "opacity":1,
         "type":"tilelayer",
         "visible":true,
         "width":10,
         "x":0,
         "y":0
        }, 
        {
         "compression":"zlib",
         "data":"eJxjYKA+YKKBmYMF0NpvAATIAAU=",
         "encoding":"base64",
         "height":10,
         "name":"Tile Layer 2",
         "opacity":1,
         "type":"tilelayer",
         "visible":true,
         "width":10,
         "x":0,
         "y":0
        }],
 "nextobjectid":1,
 "orientation":"orthogonal",
 "properties":
    {

    },
 "renderorder":"right-down",
 "tileheight":25,
 "tilesets":[
        {
         "firstgid":1,
         "image":"grass_tile.png",
         "imageheight":25,
         "imagewidth":26,
         "margin":0,
         "name":"grass_tile",
         "properties":
            {

            },
         "spacing":0,
         "tilecount":1,
         "tileheight":25,
         "tilewidth":25
        }, 
        {
         "firstgid":2,
         "image":"house2.png",
         "imageheight":70,
         "imagewidth":68,
         "margin":0,
         "name":"house2",
         "properties":
            {

            },
         "spacing":0,
         "tilecount":1,
         "tileheight":70,
         "tilewidth":68
        }],
 "tilewidth":25,
 "version":1,
 "width":10
}```
``````

**Comment by beeglebug** [comment]
There are a couple of things in that data which won't work. 
- You are using compressed layers, which this library doesn't support. You need to use csv or base64 uncompressed.
- Your image dimensions and tileset data don't line up (the image is 25x26, but the tiles are 25x25), which was causing an error. I've made some changes to allow this, as it should be a recoverable error. You will now get a warning telling you something is up, but it should load.

Can you checkout the branch `fix/#9-frame-does-not-fit` and see if it helps you?

**Comment by beeglebug** [comment]
I'm also going to update the readme to better list the tiled features which are currently supported!
